src/components/layout/_layout.scss

```diff
@@ -73,14 +78,14 @@ $layoutGapWidth: gutter(5 / 6);
 
   &__content {
     min-width: 0;
-    flex-grow: 1;
     flex-shrink: 0;
     padding: 0;
     width: 100%;
 
     @include sgBreakpoint(large-only) {
       max-width: calc(100% - #{$layoutAsideWidth + $layoutGapWidth});
       flex-basis: 1px;
+      flex-grow: 1;
```

Comment:
main content should grow only on desktop, pushing "aside content" to the right. On mobile it shouldn't grow because it will create a gap between main content and "aside content" which, on mobile, is below